doc/src/records/dhwsys.md

```diff
@@ -221,12 +397,100 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
 
 **wsTSetPointLH=*float***
 
-  Specifies the hot water setpoint temperature for all child DHWLOOPHEATERs.  Used only for HPWH-based DHWHLOOPEATERs (HPWH explicitly models tank temperatures and heating controls), otherwise has no effect.
+  Specifies the hot water set point temperature for all child DHWLOOPHEATERs.  Used only for HPWH-based DHWHLOOPEATERs (HPWH explicitly models tank temperatures and heating controls), otherwise has no effect.
 
   **Units**   **Legal Range**   **Default**   **Required**   **Variability**
   ----------- ----------------- ------------- -------------- -----------------
     ^o^F        $>$ 32 ^o^F      wsTSetPoint        No             hourly
 
+**wsTSetpointDes=*float***
+
+Specifies the design (sizing) set point temperature.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "wsTUse",
+  required: "No",
+  variability: "constant") %>
+
+**wsVolRunningDes=*float***
+
+Running volume for design. Active volume (above aquastat) equals to a full tank volume, defaults from EcoSizer at end of prerun if not input. No direct use, must be passed to DHWHEATER via ALTER.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $>$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsASHPTSrcDes=*float***
+
+Design (sizing) source air temperature for HPWH.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "Heating design temperature",
+  required: "No",
+  variability: "At the start and at the end of interval") %>
+
+**wsFxDes=*float***
+
+Excess size factor for domestic hot water design. wsFxDes is applied when wsHeatingCapDes and/or wsVolRunningDes are defaulted from EcoSizer at the end of the prerun. There is no effect if those values are input.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $>$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsDRMethod=*choice***
+
+Response control method for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
+Choice, Description
+NONE, None
+SCHED, Demand response schedule
+END
+%>
+
+<%= member_table(
+  units: "",
+  legal_range: "See table above",
+  default: "NONE",
+  required: "No",
+  variability: "At the start of a run") %>
+
+**wsDRSignal=*choice***
+
+Response control signal for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
```

Comment:
We should add descriptions for these choices too. @chipbarnaby or @pmskintner, we might need your help here since there isn't anything in the source code (that I could find).

Reply:
Here's some documentation on the commands:

ON - Normal operation following the water heater's internal control logic.
TOO - Tops of the tank once by engaging the all the available heating sources (compressor and resistive elements) in the water heater to heat the tank to setpoint, regardless of the current condition.
TOOLOR - Tops of the tank once and locks out the resistance elements, just the compressor is used to heat the tank to setpoint. 
TOOLOC- Tops of the tank once and locks out the compressor, just the resistance elements are used to heat the tank to setpoint. 
TOT - Tops of the tank on a timer using all the available heating sources (compressor and resistive elements) in the water heater. The tank starts a timer and heating to setpoint when the call is received, and repeats the heating call when the timer reaches zero.
TOTLOR - Tops of the tank on a timer and locks out the resistance elements, just the compressor is used to heat the tank to setpoint. 
TOTLOC - Tops of the tank on a timer and locks out the compressor, just the resistance elements is used to heat the tank to setpoint. 
LOC - Locks out the compressor from the water heater's normal internal control logic.
LOR - Locks out the resistive elements from the water heater's normal internal control logic.
LOCLOR  - Locks out the compressor and resistive elements from the water heater's normal internal control logic.
COUNT - Not sure on this one... @chipbarnaby?

I believe the setpoint can also be input as a schedule, i.e. Choice